scrapy/aws.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import smtplib
+from email import encoders as Encoders
+from email.mime.base import MIMEBase
+from email.mime.multipart import MIMEMultipart
+from email.mime.nonmultipart import MIMENonMultipart
+from email.mime.text import MIMEText
+from email.utils import COMMASPACE, formatdate
+from io import BytesIO
+
+from scrapy.utils.misc import arg_to_iter
+
+class SESender(object): 
+
+    def __init__(self, AWS_HOST_NAME, AWS_USER_NAME, AWS_PASSWORD, AWS_PORT_NUMBER, FROM_ADDRESS):
+        self.host_name = AWS_HOST_NAME
+        self.user_name = AWS_USER_NAME
+        self.password = AWS_PASSWORD
+        self.port_number = AWS_PORT_NUMBER
+        self.from_address = FROM_ADDRESS
+
+    def construct_message(self, to, subject, body, cc, attachments, mimetype, charset):
```

Comment:
Could some of these be assigned default values? `attachments=None` sounds like a reasonable one, for instance.